sourcegraph/sourcegraph
# Code Navigation: different icons for different file types

Previously, each file in the tree view featured a default icon, causing challenges for users trying to swiftly scan and discern the contents of the files.
![Screenshot 2023-11-17 at 11 47 55 AM](https://github.com/sourcegraph/sourcegraph/assets/62355966/a5b523f1-93f4-41b6-a936-872e3cb6bb62)

Now, files exhibit distinctive icons corresponding to their file types. Furthermore, test files are easily identifiable through the inclusion of a superimposed indicator on the bottom right corner of the corresponding icon.

Video: 
https://github.com/sourcegraph/sourcegraph/assets/129280/87210e2d-e9c5-441e-b4c5-adf16b4da46a


## Test plan
Manual testing:
1) check that every file type has the correct icon associated with it when rendered in the tree.

Ships with unit tests for helper functions
